core/src/cats/cats.h

```diff
@@ -142,8 +142,8 @@ struct JobDbRecord {
   char cRealEndTime[MAX_TIME_LENGTH]{0};
 
   // Extra stuff not in DB
-  int limit = 0;  /**< limit records to display */
-  int offset = 0; /**< offset records to display */
+  uint32_t limit = 0;  /**< limit records to display */
+  uint32_t offset = 0; /**< offset records to display */
```

Comment:
Are we certain that 32bits is enough when 31bits evidently were not ?
Probably best to change them to 64bit to be safe!